changelog.txt

```diff
@@ -13,6 +13,7 @@
 * Dev - Fix failing test cases associated with WooCommerce 10.0.x
 * Fix - Prevent multiple save appearance AJAX calls on Block Checkout
 * Fix - Fix required field error message and PHP warning for custom checkout fields that don't have a label
+* Fix - Fix fatal when processing Boleto setup intents via webhooks
```

Comment:
I think we should update the changelog to reflect that this could affect OXXO, Multibanco, and Boleto.
```suggestion
* Fix - Fix fatal when processing Boleto, OXXO, or Multibanco setup intents via webhooks
```